docs/Getting Started/RHEL and CentOS.rst

```diff
@@ -1,173 +1,165 @@
 RHEL and CentOS
 ===============
 
-`kABI-tracking
-kmod <http://elrepoproject.blogspot.com/2016/02/kabi-tracking-kmod-packages.html>`__
-or
-`DKMS <https://en.wikipedia.org/wiki/Dynamic_Kernel_Module_Support>`__
-style packages are provided for RHEL / CentOS based distributions from
-the official zfsonlinux.org repository. These packages track the
-official ZFS on Linux tags and are updated as new versions are released.
-Packages are available for the following configurations:
-
-| **EL Releases:** 6.x, 7.x, 8.x
+`DKMS`_ or `kABI-tracking kmod`_ style packages are provided for RHEL and
+CentOS based distributions from the OpenZFS repository. These packages are
+updated as new versions are released. Only the current repository for each
+major release is updated with new packages. Packages are available for the
+following configurations:
+
+| **EL Releases:** 6, 7.9, 8.2
 | **Architectures:** x86_64
 
-To simplify installation a zfs-release package is provided which
-includes a zfs.repo configuration file and the ZFS on Linux public
-signing key. All official ZFS on Linux packages are signed using this
-key, and by default yum will verify a package's signature before
-allowing it be to installed. Users are strongly encouraged to verify the
-authenticity of the ZFS on Linux public key using the fingerprint listed
-here.
+To simplify installation a *zfs-release* package is provided which includes
+a zfs.repo configuration file and public signing key. All official OpenZFS
+packages are signed using this key, and by default yum or dnf will verify a
+package's signature before allowing it be to installed. Users are strongly
+encouraged to verify the authenticity of the ZFS on Linux public key using
```

Comment:
```suggestion
encouraged to verify the authenticity of the OpenZFS public key using
```

Reply:
Maybe I'm wrong and it was intentional.

Reply:
This was actually intentional since the name on the key itself is still "ZFS on Linux" and I wanted to avoid any confusion.